.github/workflows/extension-tests.yml

```diff
@@ -55,17 +55,17 @@ jobs:
           - "8.1"
           - "8.2"
         extension-name:
-          - "phpstan-phpunit"
-          - "phpstan-strict-rules"
+          - "phpstan-phpunit:bf47c49afb7dc6e897d1f3c9b61b4bd82cc4d6e6"
```

Comment:
If this was a space instead of `:`, I think it'd be passed as two separate arguments into the bash script, and we wouldn't have to do any crazy parsing like `IFS=" " read -r -a config <<< "${1//:/ }"` (no idea what that is, you might be destroying the universe for what I know :D) and just read the other argument like we read the extension name.